infra/cifuzz/example_main.yml

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
super nit: is this empty line necessary?

Reply:
Nope, removed it.